release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Release notes
 
+## Aug 20, 2019
+
+The following updates were made in this release:
+
+**iOS Core release 2.3.3**
+
+* Reduced the number of threads being consumed by SDK and also the CPU usage of SDK.
+* Fixed a memory leak issue.
+* Added property `eventUniqueIdentifier` to the `ACPExtensionEvent` object.
```

Comment:
Please reword so that it reads as follows:

"Added the `eventUniqueIdentifier` property  to the `ACPExtensionEvent` object."